remoulade/state/backends/redis.py

```diff
@@ -43,6 +43,10 @@ def set_state(self, state, ttl):
             pipe.expire(message_key, ttl)
             pipe.execute()
 
+    def set_progress(self, message_id, progress):
```

Comment:
I would prefer you use the set this value via the set_state method. 
for the tll, we do not really care but it's always better to have a ttl to ensure keys to not stay infinitely in redis so, chnge the function definition like this.

`set_state(self, state, ttl=3600)`